Integrations/src/main/java/io/deephaven/integrations/learn/Gatherer.java

```diff
@@ -1,30 +1,55 @@
 package io.deephaven.integrations.learn;
 
 import io.deephaven.db.v2.sources.ColumnSource;
+// REMOVE THIS AFTER REMOVING makeIndexSet!
+import io.deephaven.db.tables.Table;
 
+/**
+ * Gatherer takes Deephaven columnar data and places it into a buffer to be used by Python.  The Python object will take
+ * data from the buffer and use it to construct a 2d array of specified size.
+ */
 public class Gatherer {
 
+    /**
+     * Generate an index set from a table.
+     * THIS IS JUST FOR EASY PYTHON TESTING.  REMOVE BEFORE PROD.
+     *
+     * @param t table
+     * @return index set
+     */
+    public static IndexSet makeIndexSet(final Table t) {
+        final IndexSet is = new IndexSet(t.intSize());
+
+        for(long idx : t.getIndex()) {
+            is.add(idx);
+        }
+
+        return is;
+    }
+
     /**
      * Copy data from a table into a 2d tensor of Booleans.
      *
      * @param indexSet indices of the rows of the table to put into the tensor
      * @param columnSources columns of data to put into the tensor
+     * @param transpose a boolean indicating the user's intent to return a transposed Python object
      * @return contiguous RAM allocated for the tensor. When a numpy tensor is passed in for this argument, jpy will
      *         handle passing the memory reference as a 1d java array here.
      */
-
-    public static Boolean[] booleanTensorBuffer2D(final IndexSet indexSet, final ColumnSource<?>[] columnSources) {
+    public static boolean[] tensorBuffer2Dboolean(final IndexSet indexSet,
```

Comment:
```suggestion
    public static boolean[] tensorBuffer2DBoolean(final IndexSet indexSet,
```
Camel case.  You don't need to do lower case for primitive.